env/optional.txt

```diff
@@ -0,0 +1,2 @@
+VITE_APP_AIR_TABLE_KEY
+VITE_APP_GIPHY_API_KEY
```

Comment:
VITE_APP_GIPHY_API_KEY
VITE_APP_GEYSER_NOSTR_PUBKEY

These are needed for certain operations